src/index.ts

```diff
@@ -10,10 +10,11 @@ let ldClient: LDClient;
 
 class FlagsStateInjector {
   async element(element: Element) {
-    // fetch all flag values for client-side SDKs as evaluated for an anonymous user
-    // use a more appropriate user key if needed
-    const user = { key: "anonymous" };
-    const allFlags = await ldClient.allFlagsState(user, {
+      // fetch all flag values for client-side SDKs 
+      // as evaluated for a context that has not logged in
+      // use a more appropriate context key if needed
+    const context = { key: "guest" };
```

Comment:
In our docs example, `guest` is in single quotes instead of double. Which should we use?

Reply:
Brian confirmed either works, but he prefers single quotes, so changing these to single to match the docs site.
```suggestion
    const context = { key: 'guest' };
```